extensions-core/s3-extensions/src/test/java/org/apache/druid/storage/s3/S3DataSegmentKillerTest.java

```diff
@@ -347,7 +347,7 @@ public void test_kill_listOfSegments_multiDeleteExceptionIsThrown()
       segmentKiller.kill(ImmutableList.of(DATA_SEGMENT_1, DATA_SEGMENT_2));
     }
     catch (SegmentLoadingException exc) {
-      Assert.assertEquals("Couldn't delete from bucket: [test_bucket] these files [[key1/]]", exc.getMessage());
+      Assert.assertEquals("Couldn't delete segments from s3 see logs for more details", exc.getMessage());
```

Comment:
You should use `Assert.assertThrows` rather than doing this. This approach has the problem that if `segmentKiller.kill` _doesn't_ throw an exception for some reason, then the test will still pass, because this `Assert.assertEquals` never gets called. But, we want it to fail if the expected exception isn't thrown.

Reply:
will do